README.md

```diff
@@ -167,22 +165,9 @@ asyncTask();
 
 ## Supported Data Types
 
-**TL;DR:** Currently Numbers, Strings, Typed Arrays, and common Array<T> types are supported. Returning a high-level data type from an imported JavaScript function, and better AssemblyScript Class support will be coming later. For more detailed type support information, please see the [`lib/asbind-instance/supported-ref-types.js`](./lib/asbind-instance/supported-ref-types.js) and [`test/assembly/test.ts`](./test/assembly/test.ts).
+All primitive types, ie. Numbers (`u8`, `f32`, ...) , Strings, Typed Arrays (`Uint8Array`, `Float32Array`, ...) are supported. All of those types can also be used with `Array<T>`.
```

Comment:
Awesome! I think we may want to be more clear on what is not supported (That way we don't get issues opened if we do support it).

But I don't think it should be blocking :smile: